cmd/d2/help.go

```diff
@@ -61,7 +61,7 @@ func shortLayoutHelp(ctx context.Context, ms *xmain.State) error {
 %s
 
 Usage:
-  To use a particular layout engine, set the environment variable D2_LAYOUT=[layout name].
+  To use a particular layout engine, set the environment variable D2_LAYOUT=[name] or flag --layout=[name].
```

Comment:
Not sure about conventions, but flags resonate as true/false things.
I'd go with `or arg --layout=`

Reply:
It's bizarre when you consider the meaning of flag but yea, the convention is that flag is any "tagged" argument, even if it's not really technically a flag.

Reply:
But this is only in golang world, anywhere else I believe people use `args`.
Even the function we call is `Args`, so why use `flag` here instead of `arg`?

Reply:
That's not been my experience. Args are all "arguments" after flag like `--flag`  arguments. They're either called flags or options, not just plain args. See any manpage you have installed like `man grep`, `man bash`, `man awk`, `man curl`.

Reply:
yeah, makes sense.
had the impression I always read `args` in those. 😓

Reply:
good lord the man semantic language.

Reply:
<img width="1313" alt="Screen Shot 2022-11-16 at 5 13 35 PM" src="https://user-images.githubusercontent.com/3120367/202329821-a73971f1-0a3a-47c5-98a9-1e3e6eacbbf7.png">

Reply:
Yea roff is interesting to say the least.